composer.json

```diff
@@ -16,8 +16,8 @@
     },
     "require": {
         "php": "^7.1 || ^8.0",
-        "symfony/property-access": "^3.0 || ^4.0 || ^5.0",
-        "symfony/validator": "^3.0 || ^4.0 || ^5.0",
+        "symfony/property-access": "^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/validator": "^3.0 || ^4.0 || ^5.0 || ^6.4",
```

Comment:
If it is possible please define the version as `^6.0`

Reply:
fixed